modules/core/arrow-core-data/src/main/kotlin/arrow/typeclasses/Applicative.kt

```diff
@@ -18,4 +18,9 @@ interface Applicative<F> : Apply<F> {
 
   override fun <A, B> Kind<F, A>.map(f: (A) -> B): Kind<F, B> =
     ap(just(f))
+  
+  fun <A> Kind<F, A>.replicate(n: Int): Kind<F, List<A>> =
```

Comment:
Could it be done for `Monoid`.

Reply:
You mean to combine to any monoid? Sure, do you have any use case in mind? I'd keep a list version as default tho

Reply:
I added an overload that works on any monoid, but kept the list one as I think it is the most useful one. I would have defined it in terms of the more general monoid one, but the List monoid is not available in that module

Reply:
Right, there's a circular dependency problem hahah

Thanks for the change!!